src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -117,6 +144,45 @@ object VersionInjectorPlugin extends AutoPlugin {
         gitConfFile
     }
 
+  val injectCacheKeyTask =
+    injectCacheKey <<= (resourceManaged in Compile, organization, name, version, streams, cacheKey) map {
```

Comment:
I was just following the style of injectGitTask, and injectArtifactTask. I'm sure I could use :=, but I'm not sure how it would work